donkeycar/management/base.py

```diff
@@ -252,6 +252,7 @@ def parse_args(self, args):
         parser.add_argument('--start', type=int, default=0, help='first frame to process')
         parser.add_argument('--end', type=int, default=-1, help='last frame to process')
         parser.add_argument('--scale', type=int, default=2, help='make image frame output larger by X mult')
+        parser.add_argument('--user', default=True, action='store_false', help='show user input on the video')
```

Comment:
Call this option `--draw-user-input`. `--user` is a bit too succinct.

Reply:
Done!